crates/extensions/c8y_mapper_ext/src/config.rs

```diff
@@ -87,12 +91,30 @@ impl C8yMapperConfig {
         .try_into()
         .expect("topics that mapper should subscribe to");
 
-        for topic in operations.topics_for_operations() {
-            topic_filter.add(&topic)?;
+        if let Ok(operations) = Operations::try_new(config_dir.join("operations").join("c8y")) {
+            for topic in operations.topics_for_operations() {
+                topic_filter.add(&topic)?;
+            }
         }
 
         Ok(topic_filter)
     }
+
+    /// List of all possible external topics that Cumulocity mapper addresses. For testing purpose.
+    pub fn default_external_topic_filter() -> TopicFilter {
+        vec![
+            "tedge/measurements",
+            "tedge/measurements/+",
+            "tedge/alarms/+/+",
+            "tedge/alarms/+/+/+",
+            "tedge/events/+",
+            "tedge/events/+/+",
+            "tedge/health/+",
+            "tedge/health/+/+",
```

Comment:
which component publishes the health status on `tedge/health/+/+` ?

Reply:
Look at this test https://github.com/thin-edge/thin-edge.io/blob/main/tests/RobotFramework/tests/cumulocity/service_monitoring/service_monitoring.robot#L102

Reply:
Yeah, got it. I got confused that the health topics for child devices will be like `tedge/<child-id>/health/+`.